### PR TITLE
install ks on huaweicloud ecs ha

### PR DESCRIPTION
**Pull request by LiuG-lynx**
@FeynmanZhou  please review

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
Please preview and update this guide at first, there are some broken images: https://deploy-preview-166--kubesphere-v3.netlify.app/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs/

File: content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+title: "KubeSphere 在华为云 ECS 高可用实例"
+keywords: "Kubesphere 安装， 华为云， ECS， 高可用性， 高可用性，  负载均衡器"
+description: "本教程用于安装高可用性集群"
+
+Weight: 2230
+---
+
+由于对于生产环境，我们需要考虑集群的高可用性。教你部署如何在华为云 ECS 实例服务快速部署一套高可用的生产环境
+Kubernetes 服务需要做到高可用，需要保证 kube-apiserver 的 HA ，推荐下列两种方式
+ 1. 华为云 VPC
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md:11]: 华为云负载均衡器，不是 VPC。

File: content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+keywords: "Kubesphere 安装， 华为云， ECS， 高可用性， 高可用性，  负载均衡器"
+description: "本教程用于安装高可用性集群"
+
+Weight: 2230
+---
+
+由于对于生产环境，我们需要考虑集群的高可用性。教你部署如何在华为云 ECS 实例服务快速部署一套高可用的生产环境
+Kubernetes 服务需要做到高可用，需要保证 kube-apiserver 的 HA ，推荐下列两种方式
+ 1. 华为云 VPC 
+ 2. keepalived + haproxy [keepalived + haproxy](https://kubesphere.com.cn/forum/d/1566-kubernetes-keepalived-haproxy)对 kube-apiserver 进行负载均衡，实现高可用 kubernetes 集群。
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md:12]: 云上直接用负载均衡器就行，不用推荐自建 LB，运维成本比较高

File: content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+ ```bash
+ # 在当前位置创建配置文件 master-HA.yaml |包含 KubeSphere 的配置文件
+ ./kk create config --with-kubesphere v3.0.0 -f master-HA.yaml
+--- 
+# 同时安装存储插件 (支持：localVolume、nfsClient、rbd、glusterfs)。您可以指定多个插件并用逗号分隔。请注意，您添加的第一个将是默认存储类。
+./kk create config --with-storage localVolume --with-kubesphere v3.0.0 -f master-HA.yaml
+ ```
+
+ ### 集群配置调整
+目前我是开启了全量的组件,文末也提供了自定义的方法.可默认为 false
```
Comment (FeynmanZhou) [inline content/zh/docs/installing-on-linux/public-cloud/install-ks-on-huaweicloud-ecs.md:108]: 不要出现第一人称

**Comment by LiuG-lynx** [comment]
@FeynmanZhou I have modified the above content

**Pull request merged**